### PR TITLE
Added setup.py

### PR DESCRIPTION
**Pull request by livibetter**
This is a draft, not yet ready unless nothing needs to be changed.

Here is a few notes:
- The package name is `PyClock` (from `__program__` in `pyclock.py`) and the installed script is `pyclock.py`
  
  Any changes? Make them same, or?
- No `long_description` which is displayed on PyPI, usually I will just grab `READYT.rst`, but we don't have that. It would be fine, just a blank description there.
- The code was copied and pasted from my other project, so there might be issues I didn't see.

### CODE REVIEW

File: pyclock.py [hunk 1]
```diff
@@ -7,7 +7,13 @@
...
 
 
-__description__ = ' A digital clock for the terminal '
+__program__ = 'PyClock'
+__version__ = '0.0.0'
+__description__ = 'A digital clock for the terminal'
+__author__ = 'Brandon Dreager'
+__copyright__ = 'Copyright (c) 2016 Brandon Dreager'
+__license__ = 'MIT'
+__website__ = 'https://github.com/Regaerd/PyClock'
```
Comment (bdreager) [inline pyclock.py:16]: I've seen a few project where this information is in the setup.py

Opinion?

**Comment by bdreager** [comment]
> No long_description which is displayed on PyPI, usually I will just grab READYT.rst

I'm assuming the README.md won't format correctly, right? It might be worthwhile to make a simple rst version of the readme eventually.

And the naming is good.

File: pyclock.py (hunk 1 above)
Comment (livibetter) [inline pyclock.py:16]: It's fine for both cases. But I will prefer them stay with the main script.

A good example is if you want to `--version`, you end up having a version number in two places, and I will bet there is a good chance the one in main script is forgotten. So, the package ends up with right version number on PyPI, but after the installation, you get the wrong number from the main script. Of course, the script can always query Python package database. It simply is error-prone in my opinion if put in `setup.py`.

Another good practice, at least for version number, is to have `version.py`.

Nonetheless, if you prefer having them in `setup.py`, then I will move them, I don't think duplicate them there is a good idea, the reason is as same as the version above. As times goes, one would be outdated.

**Comment by livibetter** [comment]
No, they won't. Quite different, so `README.md` won't work.

File: pyclock.py (hunk 1 above)
Comment (bdreager) [inline pyclock.py:16]: I'm convinced.
The current system is fine for now.

I like the `version.py` idea. Perhaps in the future.

**Pull request merged**